spec/requests/api/v1/items_request_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe "Items API", type: :request do 
+  before(:each) do
+    @merchant = Merchant.create(name: "Awesome Merchant") 
+    @item1 = Item.create(
+      name: "Catnip Toy",
+      description: "A soft toy filled with catnip.",
+      unit_price: 12.99,
+      merchant_id: @merchant.id
+    )
+
+    @item2 = Item.create(
+      name: "Laser Pointer",
+      description: "A laser pointer to keep your cat active.",
+      unit_price: 9.99,
+      merchant_id: @merchant.id
+    )
+
+    @item3 = Item.create(
+      name: "Feather Wand",
+      description: "A wand with feathers to entice your kitty.",
+      unit_price: 15.50,
+      merchant_id: @merchant.id
+    )
+  end
+
+  it 'can fetch all items' do
+    get '/api/v1/items'
+    
+    expect(response).to be_successful
+    expect(response.status).to eq(200)
+    
+    items = JSON.parse(response.body, symbolize_names: true)[:data]
+    expect(items).to be_an(Array)
+
+    item = items[0]
+    expect(item[:id].to_i).to be_an(Integer)
+    expect(item[:type]).to eq('item')
+
+    attrs = item[:attributes]
+
+    expect(attrs[:name]).to be_an(String)
+    expect(attrs[:description]).to be_an(String)
+    expect(attrs[:unit_price]).to be_a(Float)
+    expect(attrs[:merchant_id]).to be_an(Integer)
+  end    
+
```

Comment:
We'll also need a test for when there are no merchants, "can fetch all merchants when there are no merchants" in the merchant_request_spec can be an example but might need a refactor after this error handling lesson